testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -33,6 +33,16 @@ pub struct RunLoop {
     coordinator_channels: Option<(CoordinatorReceivers, CoordinatorChannels)>,
 }
 
+#[cfg(not(test))]
+const BURNCHAIN_POLL_TIME: u64 = 30; // TODO: this is testnet-specific
+#[cfg(test)]
+const BURNCHAIN_POLL_TIME: u64 = 1; // TODO: this is testnet-specific
+
+#[cfg(not(test))]
+const POX_SYNC_WAIT_MS: u64 = 1000;
+#[cfg(test)]
+const POX_SYNC_WAIT_MS: u64 = 0;
```

Comment:
If this value is anything other than 1000, then the block download and processing time estimations won't work correctly -- the number of samples taken is supposed to be exactly equal to the number of seconds over which the samples were taken.

Reply:
Left at 1000ms, the integration tests take about 2x as long to run -- is there a better way to disable the stalling behavior during tests?

Reply:
You can set a smaller value for `ConnectionOptions::timeout`.  This is what gets used to determine how many samples the PoX sync watchdog takes.  The units are in seconds -- you can set this to something like 3 instead of 30 for a local test.

Reply:
Made this configurable via 33ec31b39